<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitWidget</class>
 <widget class="QDialog" name="GitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>List files:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonInit">
         <property name="text">
          <string>Init</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStatus">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCommit">
         <property name="text">
          <string>Commit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPush">
         <property name="text">
          <string>Push</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="Commit">
       <property name="placeholderText">
        <string>Text Commit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="Log"/>
   </item>
   <item>
    <widget class="QPushButton" name="Button_OK">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCommit</sender>
   <signal>clicked()</signal>
   <receiver>GitWidget</receiver>
   <slot>gitCommit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_OK</sender>
   <signal>clicked()</signal>
   <receiver>GitWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gitCommit()</slot>
 </slots>
</ui>
